spec/pubid_ieee/identifier_spec.rb

```diff
@@ -756,6 +756,34 @@ def context_description(ex)
 
       it_behaves_like "converts pubid to pubid"
     end
+
+    context "ISO/IEC FDIS P15289, April 2014" do
+      let(:original) { "ISO/IEC FDIS P15289, April 2014(E)" }
+      let(:pubid) { "ISO/IEC FDIS P15289, April 2014" }
+
+      it_behaves_like "converts pubid to pubid"
+    end
+
+    context "ISO/IEC/IEEE P26514/FDIS, August 2021" do
+      let(:original) { "ISO/IEC/IEEE P26514/FDIS, August 2021" }
+      let(:pubid) { "ISO/IEC/IEEE FDIS P26514, August 2021" }
+
+      it_behaves_like "converts pubid to pubid"
+    end
+
+    context "ISO/IEC/IEEE P15288-DIS-1403" do
+      let(:original) { "ISO/IEC/IEEE P15288-DIS-1403" }
+      let(:pubid) { "ISO/IEC/IEEE DIS P15288-1403" }
```

Comment:
Part of me thinks that:
* the ISO version would be "ISO/IEC/IEEE DIS 15288-1403" (no P)
* the IEEE version would be "ISO/IEC/IEEE P15288-1403/DIS" (with P, and move "DIS" to a draft stage)

But I'm happy with what you have for now...